verify/src/versioned.rs

```diff
@@ -75,10 +75,7 @@ pub fn requires_type(version: Version, method_name: &str) -> Result<bool> {
             ))),
     };
 
-    let requires = match method.ret {
-        Some(Return::Type(_)) => true,
-        _ => false,
-    };
+    let requires = matches!(method.ret, Some(Return::Type(_)));
```

Comment:
Can you make this a separate patch please. Its a bit more work but good practice in writing commit logs and explaining your changes. Gives us reviewers confidence you know what you are doing and not just following the compiler blindly.

Thanks